CHANGELOG.rst

```diff
@@ -1,5 +1,39 @@
 .. _changelog:
 
+0.31.0
+------
+
+Renku ``0.31.0`` introduces the compute resource control (CRC) service. This feature enables Renku administrators to manage the users' access to specific computing resources. The service exposes an HTTP API for the administrators to interact with. In this way, Renku administrators can create, update or delete resource pools, and can add or remove users to resource pools. Thus, this release includes new features in Renku individual components related to the compute resource service, and supports common R file extensions.
```

Comment:
This should also say something about what it means for users, i.e that they will from now on need to select a resource class not individually select compute requirements.

Reply:
I expanded the description to mention this.